release/nrc/str.sencoten/source/Makefile

```diff
@@ -0,0 +1,10 @@
+# How to build the wordlist!
+#
+# Sources:
+#  - SaanichWordFreq.txt: Provided by Dr. Timothy Montler.
+#    NOT included in this repository!
+#    SHA384 sum: 5e31a32eaab3aa23e029f72fb3097f9fb97982fe9c927dd99093d3f1071c1b81d43926e4c9abc79a1afe75c63cfb599c
+SOURCES = SaanichWordFreq.txt
```

Comment:
If the source material is not available, then the Makefile doesn't really belong in the repo either -- it adds no value to other maintainers and it'd be better to keep that with the source data. And I guess that is also true of the fix-wordlist.py file?

The problem is that including these files puts the burden of maintaining them onto the repository maintainers -- but they have no ability to test the veracity of them or to do anything useful with them. If someone else (who presumably has access to the source data) submits a PR that updates them, the maintainers cannot actually test anything about the files so it becomes difficult to know whether to accept or reject the change.

If you do decide you want to keep the files in the repo, can you put them instead into `extra/` instead of `source/`, so it can be clear that they are not source files that anyone else can do anything useful with.

Reply:
Okay! I'll put it in `extra/` It's useful for me, as a maintainer of the model :)